.chalice/config.json

```diff
@@ -3,12 +3,26 @@
   "app_name": "fusillade",
   "stages": {
     "dev": {
-      "api_gateway_stage": "api"
+      "tags": {
+        "project": "dcp",
+        "owner": "test@email.com",
+        "env": "dev"
+      },
+      "api_gateway_stage": "dev",
+      "environment_variables": {
+        "API_DOMAIN_NAME": "auth.dev.data.humancellatlas.org",
+        "DEBUG": "1",
+        "FUS_ADMIN_EMAILS": "test@email.com",
+        "FUS_DEPLOYMENT_STAGE": "dev",
+        "FUS_SECRETS_STORE": "dcp/fusillade",
+        "OPENID_PROVIDER": "humancellatlas.auth0.com"
+      }
     }
   },
-  "environment_variables": {
-    "OPENID_PROVIDER": "humancellatlas.auth0.com"
-  },
-  "lambda_timeout": 30,
-  "lambda_memory_size": 1536
+  "lambda_timeout": 10,
+  "lambda_memory_size": 1536,
+  "tags": {
+    "managedBy": "chalice",
+    "Name": "fusillade"
```

Comment:
Name is a reserved term in tagging context see https://github.com/HumanCellAtlas/dcp-community/blob/rfc-resource-tagging/rfcs/text/0006-resource-tagging.md#aws

Service would be more appropriate, this could bemoved to the build_chalice_config.sh file

Reply:
This was intended to be the `Name` mentioned in the RFC. Would "fusillade-api-{stage}" be more appropriate?

Reply:
@Bento007 Yes i think that would be appropriate; on deployment time it can get updated if there are additional lambda's applied.